.gn

```diff
@@ -140,6 +140,10 @@ exec_script_whitelist =
       "//build_overrides/build.gni",
     ]
 
+# The python interpreter to use by default. On Windows, this will look
+# for python3.exe and python3.bat.
+script_executable = "python3"
```

Comment:
Is this tested on the ubuntu docker?

Reply:
No, but Dockerfile has python3 and python-is-python3 installed. depot_tools also includes python3.